docs/polygon_draw.js

```diff
@@ -349,15 +351,20 @@ function setUpPolygonDrawing(map, firebasePromise) {
  * Adds a listener to display notes on pop-up.
  *
  * @param {google.maps.Map} map Map to display regions on
- * @param {Promise<any>} firebasePromise Promise that will complete when
- *     Firebase authentication is finished
+ * @param {Promise<any>} firebasePromise Promise with Firebase damage data (also
+ *     implies that authentication is complete)
  * @return {Promise} promise that is resolved when all initialization is done
  *     (only used by tests).
  */
 function processUserRegions(map, firebasePromise) {
```

Comment:
maybe update function name and doc? + maybe a '.' a the end of the firebasePromise param?

Reply:
I don't think we have a convention of ending parameter docs with periods (see the previous line). Updated doc and renamed.

Reply:
It's just so similar to the comment below, which does end in a period, but I won't die on this hill.

Reply:
Removed period from @return statement :)